item-seller/packages/contracts/.env

```diff
@@ -20,11 +20,14 @@ CHAIN_ID=31337
 # RPC_URL=https://devnet-game-sync.nursery.reitnorf.com
 
 #SMART STORAGE UNIT ID
+#Copy this info from in game smart storage unit
 SSU_ID=06496282436511424955311689125460433858625928370621647559384988961918634545666
 
+
 ##### ITEM SELLER CONFIGURATION
-#ITEM ID 77800 - Common Ore
-INVENTORY_ITEM_ID=1235
+# Its a prerequisite to have the item deposited into the SSU
+# Once deposited get the itemId from https://blockchain-gateway-nova.nursery.reitnorf.com/smartassemblies/<ssu_id>
```

Comment:
Needs clearer explanation - users should get the `itemId` from the corresponding item in the `inventory.storageItems` array of their SSU at that address.

Reply:
Yes adding that along with the screenshots in docs